### DISCUSSION TITLE
Yolov3 vs Yolov8

### DISCUSSION THREAD

**Discussion by irum** [ultralytics/ultralytics#2409] [Q&A]
Please can you give me a comparison of both versions and why researchers are still using yolov3?
Thank you

**Comment by glenn-jocher** [comment]
@irum hello there! 👋

YOLOv3 and YOLOv8 are both part of the YOLO (You Only Look Once) family of object detection models, but they have significant differences.

YOLOv3 is an earlier version that was widely adopted due to its good balance of speed and accuracy. It's still used by researchers and developers for several reasons, including compatibility with legacy systems, familiarity, and sufficient performance for certain applications.

YOLOv8, on the other hand, represents the latest advancements in the YOLO series. It offers improved accuracy, speed, and features such as support for multiple computer vision tasks like detection, segmentation, and pose estimation. YOLOv8 also includes enhancements in model architecture and training techniques, leading to better performance on a wide range of datasets.

Researchers might still use YOLOv3 for benchmarking, educational purposes, or specific use cases where the model's characteristics align well with their project requirements. However, for those looking to leverage the latest technology for state-of-the-art performance, YOLOv8 is the recommended choice.

For more detailed information on YOLOv8 and its capabilities, you can visit our documentation at https://docs.ultralytics.com. 😊
